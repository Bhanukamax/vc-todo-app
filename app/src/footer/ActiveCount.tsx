import React from "react";
import styled from "styled-components";

interface IProps {
  count: string;
}

export const ActiveCount: React.FC<IProps> = (props: IProps) => {
  return <Count>{props.count} active items</Count>;
};

const Count = styled.p`
  font-style: normal;
  font-weight: normal;
  font-size: 1em;
  line-height: 25px;
  color: #444;
`;