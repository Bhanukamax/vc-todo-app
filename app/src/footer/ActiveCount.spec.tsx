import React from "react";
import { shallow } from "enzyme";
import { ActiveCount } from "./ActiveCount";

describe("<ActiveCount/>", () => {
  it("renders correctly", () => {
    const component = shallow(<ActiveCount />);

    expect(component).toMatchSnapshot();
  });

  it("reders correctly with props", () => {
    const component = shallow(<ActiveCount count={5} />);

    expect(component).toMatchSnapshot();
  });
});
