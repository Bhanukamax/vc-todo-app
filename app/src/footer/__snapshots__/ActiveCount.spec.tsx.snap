// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<ActiveCount/> reders correctly with props 1`] = `
<styled.p>
  5
   active items
</styled.p>
`;

exports[`<ActiveCount/> renders correctly 1`] = `
<styled.p>
   active items
</styled.p>
`;
